Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -68,6 +68,16 @@ class MainTabBarViewController: UITabBarController {
             selector: #selector(updateTabs),
             name: .updatedExperemental,
             object: nil)
+
+        NotificationCenter.default.addObserver(
+                self,
+                selector: #selector(appMovedToBackground),
+                name: UIApplication.willResignActiveNotification,
+                object: nil)
+    }
+
+    @objc func appMovedToBackground() {
+        App.shared.intercomConfig.hide()
```

Comment:
This seems a bit more complicated than needed: will it work if we put this line into SceneDelegate.swift sceneWillResignActive() method? Then no need to use notifications